pkg/core/consensus/score/testutil.go

```diff
@@ -18,9 +18,7 @@ import (
 // Helper for reducing test boilerplate
 type Helper struct {
 	*consensus.Emitter
-	lock               sync.RWMutex
-	failOnFetching     bool
-	failOnVerification bool
```

Comment:
please rebase this commit with the previous one

Reply:
done